db/seeds.rb

```diff
@@ -46,26 +46,26 @@ def build_orders
-    item7 = Item.create(title: "Queso Quesadillas", description: "These delicious veggie dillas can be frisbeed to your door upon request.", price: 550, image:"quick_quesadillas.png")
-    item8 = Item.create(title: "Death by Burrito", description: "That's no moon, that's a Beef Burrito smoothered in spicy green chili sauce!", price: 750, image:"death_by_burrito.png")
-    item9 = Item.create(title: "Flaming Fajitas", description: "These veggie fajitas will set your mouth on fire.", price: 1200, image:"flaming_fajitas.png")
-    item10 = Item.create(title: "Breakfast Quesadilla", description: "One deliscious egg, cheese, and potatoe dilla.", price: 650, image:"breakfast_dilla.png")
-    item11 = Item.create(title: "Halo", description: "All fresh vegetables which gives you the moring shine", price: 500, image:"burrito_360.png")
-    item12 = Item.create(title: "Vegi delight", description: "Eat all your vegies, tomatoes, red onions, green pepers, mushrooms and more)", price: 700, image:"burrito_360.png")
-    item13 = Item.create(title: "Not too Sweet", description: "Just enough to satisfy for sweet quench, all sweet things", price: 500, image:"burrito_360.png")
-    item14 = Item.create(title: "Classic", description: "Our same classic burrito with it's authentic flavors", price: 800, image:"burrito_360.png")
-    item15 = Item.create(title: "Hot n Spicy", description: "All the hot and spicy things you can handle", price: 1000, image:"burrito_360.png")
-    item16 = Item.create(title: "Tamale Torpedo", description: "These little tamale bombs come in one flavor: spicy!", price: 400, image:"burrito_360.png")
-    item17 = Item.create(title: "Rice and Bombs", description: "Our slow-simmered black beans and cilantro-lime rice packaged up in individual parachute packs.", price: 300, image:"rice_bombs.png")
-    item18 = Item.create(title: "Hot Sauce Bucket", description: "Just what it sounds like: a small bucket of our house hot sauce.", price: 300, image:"sauce_bucket.png")
-    item19 = Item.create(title: "Chips and Salsa", description: "A bag of chips bubble wrapped for special delivery with a small bucket of our house hot sauce.", price: 400, image:"chips_salsa.png")
-    item20 = Item.create(title: "Enola Empenada", description: "This empenada is filled with our famous mushroom salsa.", price: 500, image:"girl_from_empenada.png")
+    item1 = Item.create(title: "Gut Buster", description: "If its in the store, it's in this burrito!", price: 1000, image: open("app/assets/images/burrito_360.png"))
+    item2 = Item.create(title: "Atomic Sunrise", description: "Spicy green chili, egg, and potatoe burrito.", price: 850, image: open("app/assets/images/burrito_360.png"))
+    item3 = Item.create(title: "Fiesta Especial", description: "A 6 foot party sized burrito to share with your family and friends.", price: 10000, image:open("app/assets/images/burrito_360.png"))
```

Comment:
Useless assignment to variable - `item3`.<br>Line is too long. [196/80]<br>Space missing after colon.